entries/t/telekom.de.json

```
@@ -9,6 +9,7 @@
     "categories": [
       "utilities"
     ],
+    "notes": "Mobile phone number required as fallback when enabling authenticator app.",
```

I suggest this rephrasing as it will better align with the notes for sites with similar requirements.
```suggestion
    "notes": "Software 2FA requires setting up SMS 2FA as a fallback.",
```